recipes/bob.blitz/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set version = "2.0.8" %}
+
+package:
+  name: bob.blitz
+  version: {{ version }}
+
+source:
+  fn: bob.blitz-{{ version }}.zip
+  url: https://pypi.python.org/packages/source/b/bob.blitz/bob.blitz-{{ version }}.zip
+  md5: 1d5db49ce0a5f64b8bd729e0455f52ab
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python -B setup.py install --single-version-externally-managed --record
+    record.txt
+
+requirements:
+  build:
+  - python
+  - setuptools
+  - bob.extension
+  - numpy
```

Comment:
Presently, that syntax applies only to numpy.  There is a lot of discussion on the topic - most recent at https://github.com/conda/conda-build/pull/848